@here/harp-mapview/lib/MapView.ts

```diff
@@ -1997,6 +2015,58 @@ export class MapView extends THREE.EventDispatcher {
         this.m_textElementsRenderer.clearOverlayText();
     }
 
+    // tslint:disable: max-line-length
+    /**
+     * Adjusts the camera to look at a given geo coordinate with tilt and heading angles.
+     *
+     * #### Note on `target` and `bounds`
+     *
+     * If `bounds` are specified, `zoomLevel` and `distance` parameters are ignored and `lookAt`
+     * calculates best zoomLevel (and possibly target) to fit given bounds.
+     *
+     * Following table shows how relation between `bounds` and target.
+     *
+     * | `bounds`             | `target`    | actual `target`
+     * | ------               | ------      | --------
+     * | [[GeoBox]]           | _ignored_   | `bounds.center` is used as new `target`
```

Comment:
Why is the target ignored in this case?

Reply:
I would expect the same behavior as for `GeoCoordLike[]` below, i.e. if target is undefined it will use the GeoBox.center if target is defined it will use it and zoom out so that the geobox is visible

Reply:
Agree, that was unintuitive. Now `target` has priority and only if it's not specified, we derive it from  `bounds`.